spec/factories/move.rb

```diff
@@ -155,6 +154,8 @@
     association(:profile)
     association(:from_location, :prison, factory: :location)
     association(:to_location, :court, factory: :location)
+    association(:supplier)
```

Comment:
Isn't this duplicated from line 15? Looks like every one of these factories now includes a supplier (which also might be kind of expensive and affect the test suite performance).

Reply:
Doesn't use the other factory as a parent so I guess it's needed.

Reply:
yes, I tried to remove it, but test didn't pass

Reply:
Okay, thanks for the explanation.